test/fit_tests/tests/rackhd11/test_rackhd11_api_skupack.py

```diff
@@ -18,7 +18,7 @@
 
 # Select test group here using @attr
 from nose.plugins.attrib import attr
-@attr(all=True, regression=True, smoke=True)
+@attr(all=True, regression=False, smoke=False)

```

Comment:
@nortonluo I thin you need to change the all=True to False.  Also, how about another tag for firmware - we could use that to find all firmware tests?  firmware=True.